server/channels/api4/file.go

```diff
@@ -41,7 +41,7 @@ func (api *API) InitFile() {
 
 	api.BaseRoutes.Team.Handle("/files/search", api.APISessionRequiredDisableWhenBusy(searchFiles)).Methods("POST")
 
-	api.BaseRoutes.PublicFile.Handle("", api.APIHandler(getPublicFile)).Methods("GET")
+	api.BaseRoutes.PublicFile.Handle("", api.APIHandler(getPublicFile)).Methods("GET", "HEAD")
```

Comment:
If it's a HEAD request we shouldn't serve a response body, right?

Reply:
We don't. Go takes care of it.

Reply:
To expand, `http.ServeContent` checks the HTTP method and then only responds with the headers. https://cs.opensource.google/go/go/+/refs/tags/go1.21.6:src/net/http/fs.go;l=352-355